packages/tenderly-core/src/internal/cli/commands/LoginCommand.ts

```diff
@@ -2,39 +2,49 @@ import open from "open";
 import axios from "axios";
 import prompts from "prompts";
 import commander from "commander";
+import { logger } from "../../../utils/logger";
 
 import { isAccessTokenSet, setAccessToken } from "../../../utils/config";
 import { TENDERLY_API_BASE_URL, TENDERLY_DASHBOARD_BASE_URL } from "../../../common/constants";
 
 export const LoginCommand = new commander.Command("login").description("login to Tenderly").action(async () => {
+  logger.info("Making LoginCommand...");
+
   if (isAccessTokenSet()) {
+    logger.debug("Access token is set. Prompting user to overwrite access token with a new one.");
     const response = await prompts({
       type: "confirm",
       name: "overwrite",
       message: "Access token already set. Would you like to overwrite it?",
     });
     if (!response.overwrite) {
+      logger.debug("User didn't request an overwrite of the token. Login command is made with existing token.");
       return;
     }
   }
 
+  logger.info("Access token isn't set. Prompting user to enter access token...");
   const accessToken = await promptAccessToken();
 
+  logger.debug("User entered access token. Setting access token...");
   setAccessToken(accessToken);
-  console.log("Successfully logged in.");
+
+  logger.info("Successfully logged in.");
 });
 
 async function promptAccessToken(): Promise<string> {
-  console.log(`Redirecting to ${TENDERLY_DASHBOARD_BASE_URL}/account/authorization`);
+  logger.debug(`Redirecting to ${TENDERLY_DASHBOARD_BASE_URL}/account/authorization`);
   await open(`${TENDERLY_DASHBOARD_BASE_URL}/account/authorization`);
 
+  logger.info("Prompting user to enter access token");
   const response = await prompts({
     type: "text",
     name: "accessToken",
     message: "Enter access token",
     validate: validator,
   });
 
+  logger.info("User entered access token:", response.accessToken);
```

Comment:
We can't log a user access token. Can we change the message to something like "User entered access token" without `: <accessToken>`